website/docs/integrator-section/verifiable-document/did/signing-document.md

```diff
@@ -13,16 +13,16 @@ open-attestation sign ./wrapped-documents-did --od ./signed-documents-did --publ
 
 In the example above:
 
-- `public-key` is the public key used [to bind the wallet address with the DNS location](/docs/integrator-section/verifiable-document/did/dns)
-- `key` is the wallet private key, generated when [creating the wallet](/docs/integrator-section/verifiable-document/did/create)
+- `public-key` is the public key used to [bind the wallet address with the DNS location](/docs/integrator-section/verifiable-document/did/dns).
+- `key` is the wallet private key, generated when [creating the wallet](/docs/integrator-section/verifiable-document/did/create).
 
 ```text
 ✔  success   Signed documents saved to ./signed-documents-did
 ```
 
 ## Verifying the documents
 
-Head to `dev.tradetrust.io` and drag and drop one of the issued /signed documents. The document will be verified, then displayed.
+Head to `dev.tradetrust.io` and drag and drop one of the issued or signed documents. The document will be verified then display.
```

Comment:
```suggestion
Head to `dev.tradetrust.io` and drag and drop one of the issued or signed documents. The document will be verified then displayed.
```

Reply:
I saw in many product documentation, 
it's more common to use "a message (or dialog) displays" or "a message shows up", 
instead of "a message is displayed". 
The active voice is also recommended, instead of the passive voice...

Reply:
It looks like the main issue with this line is, the mixed use of active & passive voices.
Suggest the following: 
> After being verified, the document will display.